src/invite/Invite.utils.tsx

```diff
@@ -0,0 +1,8 @@
+import dynamicLinks from '@react-native-firebase/dynamic-links'
```

Comment:
there's some strong sentiment among the team that we should stop creating util files because they become a dumpling ground for poorly thought out abstractions. this is not the case here but i would suggest to not create a new utils file for this...

Reply:
OK - where would you like this logic to live instead?

Reply:
if you go down the route generating this link and storing in redux, then maybe in the firebase saga?